documentation/staging/content/userguide/base-images/custom-images.md

```diff
@@ -12,29 +12,26 @@ description: "Create custom images and apply patches."
   - [Create a custom base image](#create-a-custom-base-image)
   - [Create a custom image with your domain inside the image](#create-a-custom-image-with-your-domain-inside-the-image)
   - [Create a custom image with your model inside the image](#create-a-custom-image-with-your-model-inside-the-image)
-- [Apply patched images to a running domain](#apply-patched-images-to-a-running-domain)
-  - [Domain in PV](#domain-in-pv)
-  - [Model in Image with auxiliary images](#model-in-image-with-auxiliary-images)
-  - [Model in Image without auxiliary images](#model-in-image-without-auxiliary-images)
-  - [Domain in Image](#domain-in-image)
 
 ### Use the WebLogic Image Tool to create custom images
 
-To create custom images for your domain resource, you can use the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/).
-Download and install the WebLogic Image Tool (WIT) following the WIT [Setup](https://oracle.github.io/weblogic-image-tool/userguide/setup/) instructions.
+You can use the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/) (WIT) to build your own WebLogic Server or
+Fusion Middleware Infrastructure images (with the latest Oracle Linux images, Java updates, and WebLogic Server patches),
+apply one-off patches to existing OCR images, or overlay your own files and applications on top of an OCR image.  
+
+Download and install the WebLogic Image Tool (WIT) following the
+WIT [Setup](https://oracle.github.io/weblogic-image-tool/userguide/setup/) instructions.
 Also, refer to the WIT [Quick Start](https://oracle.github.io/weblogic-image-tool/quickstart/quickstart/) Guide.
```

Comment:
The sample usages throughout this doc assume that you have installed WIT to `/tmp/imagetool` but you are of course free to install it into any location.

Reply:
(Maybe start the above paragraph with this new sentence. Or end it.)

Reply:
A similar sentence would also be useful in the step-by-step instructions below -- in the first step that now points to the WIT doc install.

Reply:
I added the info you suggested after the "Download and install the WebLogic Image Tool (WIT)... sentences. The step-by-step instructions already have this information.

Reply:
Maybe we are looking at different doc? The step-by-step do not say anything about the subsequent use of '/tmp/imagetool' as the sample location. The first step of the step-by-step instructions now point to the WIT documentation, which makes no comment about the location - instead it just says "Unzip the downloaded ZIP file into a directory of your choice".

Reply:
The step-by-step instructions I thought you were referring to are here, under "Create a custom image with your domain inside the image"
Domain in Image using WDT:
Here we explore a step-by-step approach for Domain in Image using WebLogic Deploy Tool models to create the domain home. These steps stage files to /tmp/dii-wdt-stage, assume the operator source is in /tmp/weblogic-kubernetes-operator, **assume you have installed WIT in /tmp/imagetool**, and generate a Domain in Image image named my-dii-wdt:v1.

And here, Domain in Image using WLST:
Here is a step-by-step approach for Domain in Image images using WLST. These steps stage files to dii-wlst-stage, put the domain home inside the image at /u01/oracle/user_projects/domains/dii-wlst, assume the operator source is in /tmp/weblogic-kubernetes-operator, **assume you have installed WIT in /tmp/imagetool**, and name the final image my-dii-wlst:v1.